pages.it/linux/apt-key.md

```diff
@@ -0,0 +1,23 @@
+# apt-key
+
+> Servizio di gestione delle chiavi per il gestore di pacchetti APT su Debian ed Ubuntu.
+
+- Elenca le chiavi fidate:
```

Comment:
Not sure whether "fidate" is the right translation here. I would suggest using "attendibile"

Reply:
Not sure either. `Trusted certificates` are translated to `certificati attendibili` for sure.
But I checked the Italian translation of seahorse's(the gnome application) GUI, and it uses `Mostra fidati`.
Another word could be `affidabili`.

Reply:
I checked the Italian version of `apt-key` manpages and it uses "fidati"

Reply:
good to know, thanks for checking!